scripts/install-go.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+export DEBIAN_FRONTEND=noninteractive
+VERSION=1.4.3
+OS=linux
+ARCH=amd64
+
+prefix=/usr/local
+
+which wget || apt-get install -y wget
+# not essential but go get depends on it
+which git || apt-get install -y git
+
+# grab go
+if ! [ -e "$prefix/go" ]; then
+    if ! [ -e "go$VERSION.$OS-$ARCH.tar.gz" ]; then
+        wget -q https://storage.googleapis.com/golang/go$VERSION.$OS-$ARCH.tar.gz
+    fi
+    tar -C $prefix -xzf go$VERSION.$OS-$ARCH.tar.gz
```

Comment:
This probably requires some checks if there are 'vet' and 'cover' tools too. IIRC, 1.4.3 release is botched, as google forgot to include these tools in the tarball, so we will need to use `go get` to get them. See https://github.com/coreos/rkt/blob/5084bb5d69ae65edd8acd88ef56b1f4198261c5e/tests/install-deps.sh#L32